internal/prober/multihttp/script.tmpl

```diff
@@ -67,19 +68,24 @@ function assertHeader(headers, name, matcher) {
 
 export default function() {
 	let response;
+	let url;
 	const vars = {};
 
 	{{ range .Entries }}
 	{{- $url := buildUrl .Request }}
+	{{- $urlString := buildUrlString .Request }}
+	{{- $queryParams := buildQueryParams .Request }}
 	{{- $method := .Request.Method.String }}
 	{{- $body := buildBody .Request.Body }}
 	{{- $headers := buildHeaders .Request.Headers .Request.Body }}
-	response = http.request('{{ $method }}', '{{ $url }}', {{ $body }}, {
+	url = {{ $url }}
```

Comment:
I know javascript is not my thing, but shouldn't this be `let url = {{ $url }}`?

Reply:
or possibly const?

Reply:
It's declared on line 71 already, so this line is just overwriting the reference, not creating a new one. And that's also why we can't use `const`, because the thing that makes it `const` is that you can't reassign it's value

Reply:
To be clear, I wouldn't actually write JS this way, but otherwise we'd need to give the url variable a unique name for each request